We already have an assertion (properties) that use camelCase and works well but in the element `boxshadow` but if we use this same element with camelCase is crashing as error below:

``` js
 ✖ should assert with any order of value 
         Electron 1.6.6 (Linux 0.0.0)
       AssertionError: expect div to have {boxShadow: #F00 0px 0px 10px 0px}, is receiving {boxShadow: rgb(255, 0, 0) 0px 0px 10px 0px}
           at Context.it (/tmp/sources/index.spec.js:95:0 <- /tmp/04f59fc04a48ba6526ff2c039b277c88.browserify:22734:31)
```

Please, check it.

--- a/sources/index.js
+++ b/sources/index.js
@@ -28,9 +28,10 @@ function chaiStyle(chai, utils) {
     this.assert(assertion, throwMessage, throwMessageNegative, value)
 
     function compareCSSValue(computed, expected) {
+      const propertyHifenCase = property.replace(/[A-Z]/g, (match) => '-' + match.toLowerCase())
       const fake = document.createElement('div')
       fake.style.fontSize = style.fontSize
-      fake.style.setProperty(property, expected, 'important')
+      fake.style.setProperty(propertyHifenCase, expected, 'important')
       const iframe = document.createElement('iframe')
       iframe.style.visibility = 'hidden'
       document.body.appendChild(iframe)
@@ -41,8 +42,6 @@ function chaiStyle(chai, utils) {
       const hasAutoValue = value.includes('auto')
       const reg = new RegExp(value.replace(/auto/g, '(\\d+(.\\d+)?px|auto)'))
 
-      // console.log(`${computed} === ${value}`, computed === value)
-      // console.log('expected', reg.toString(), reg.test(computed))
       return hasAutoValue
         ? reg.test(computed)
         : computed === value

--- a/sources/index.spec.js
+++ b/sources/index.spec.js
@@ -88,6 +88,10 @@ describe('chai-style', () => {
     })
 
     it('should assert with any order of value', () => {
+      expect(element).to.have.style('box-shadow', '#F00 0px 0px 10px 0px')
+    })
+
+    it('should assert with any order of value in camelCase property', () => {
       expect(element).to.have.style('boxShadow', '#F00 0px 0px 10px 0px')
     })
   })

